test/props.test.js

```diff
@@ -36,4 +36,38 @@ describe("props", () => {
       });
     });
   });
+
+  describe("EVENT_PROPS", () => {
+    it("contain all DisplayObject events", () => {
```

Comment:
`it("should be a list containing all DisplayObject events", () => {`

Reply:
got it